dts/bindings/lora/seed,wm1110-radio.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2025 Hasan Albinsaid (mail@hasanabs.com)
+# SPDX-License-Identifier: Apache-2.0
+
+description: Semtech LR1110 LoRa transceiver on Seed WM1110
+
+compatible: "seed,wm1110-radio"
```

Comment:
No, if this is a LR1110 then the compatible would be for that which has nothing to do with seed.